tests/Microsoft.Identity.Test.Unit.net45/WebUITests/DefaultOsBrowserWebUiTests.cs

```diff
@@ -107,22 +122,46 @@ public void ValidateRedirectUri()
             IWebUI webUi = new DefaultOsBrowserWebUi(_platformProxy, _logger, tcpInterceptor);
 
             // Act
-            webUi.ValidateRedirectUri(new Uri("http://localhost:12345"));
-            webUi.ValidateRedirectUri(new Uri("http://127.0.0.1:54321"));
-
-            AssertInvalidRedirectUri(webUi, "http://localhost"); // no port
-            AssertInvalidRedirectUri(webUi, "http://localhost:0"); // no port
-            AssertInvalidRedirectUri(webUi, "http://localhost:80"); // default port
+            webUi.UpdateRedirectUri(new Uri("http://localhost:12345"));
+            webUi.UpdateRedirectUri(new Uri("http://127.0.0.1:54321"));
 
+            AssertInvalidRedirectUri(webUi, "https://localhost"); // https
```

Comment:
>AssertInvalidRedirectUri(webUi, "https://localhost"); // https [](start = 12, length = 62)

I would assume that this should be ok

Reply:
Can't generate a port (well, we can, AAD won't allow it).